tests/cli/dorq/test_cli_logs.py

```diff
@@ -0,0 +1,66 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+"""
+Tests for ``orquestra.sdk._base.cli._corq._cli_logs``.
+"""
+import subprocess
+import sys
+import os
+from pathlib import Path
+
+import pytest
+
+
+@pytest.mark.slow()
+class TestConfigureVerbosenessIfNeeded:
+    class TestIntegration:
+        """
+        Python's ``logging`` is notoriously difficult to reason about, so unit tests
+        with mocks don't make much sense. Testing logging configuration with ``pytest``
+        isn't straightforward either because ``pytest`` intercepts log entries. This is
+        why we test the logging config via subprocess & stdout.
```

Comment:
suggestion: sub-process tests are notoriously difficult to debug tho. Could you try to use other means of testing?

Reply:
Unfortunately I don't know a good way to implement a test like this in a single process. I tried with capsys (https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/ae1cdfbc9e32992281540e74212883b4eacc01da/tests/cli/dorq/test_cli_logs.py#L70-L86) but the stdout/err isn't produced. [Pytest has utils](https://docs.pytest.org/en/7.1.x/how-to/logging.html) for `logging` but they serve a different purpose. Here, I want to ensure we configure the loggers hierarchy correctly, whereas `caplog` allows you to assert that a given function emitted a log record.